projects/distributed-tracing/src/shared/dashboard/dashboard-wrapper/navigable-dashboard.component.ts

```diff
@@ -84,6 +84,10 @@ export class NavigableDashboardComponent implements OnChanges {
     }
 
     if (changeObject.filterConfig) {
+      if (changeObject.filterConfig.currentValue?.implicitFilters) {
```

Comment:
I think we should likely drop the condition - we can always refresh filters if the filter config changed - it may refresh in certain situations where it's unnecessary, but it's likely rare enough to not matter. In this form, it would miss the case where filters are cleared (implicitFilters goes from defined to undefined)

Reply:
I agree.